package.json

```diff
@@ -7,7 +7,7 @@
   "author": "EDA",
   "license": "ISC",
   "scripts": {
-    "start": "webpack && node server",
+    "start": "node server && webpack --watch",
```

Comment:
Currently your changes are making `dev` and `start` scripts the same.
Please change this back, `start` script is not supposed to watch for changes. It is used as a run once script. For watching and continuios restarting of the server please use `dev` script.

Reply:
applyied changes to the ```start``` script